aten/src/ATen/native/Blas.cpp

```diff
@@ -231,51 +237,194 @@ Tensor vdot(const Tensor &self, const Tensor &other){
 
 }
 
+/*
```

Comment:
I think this should all be pulled out into an `aten/src/ATen/native/cpu/ScaledBlas.cpp` to match the CUDA & XPU implementations - I'd like to keep all of the complexity of scaled gemm ops in their own files as much as possible.